### ISSUE TITLE
Add option to always load app tabs

### ISSUE THREAD

**Issue by hackel** [philikon/BarTab#100]
App tabs should always be loaded and available.  Unfortunately, whitelisting hostnames for each app tab is not only time-consuming, it is not ideal, because I very often have additional tabs open to that particular host that are not app tabs and so do not need to be auto-loaded.  I would love to see this feature implemented.

**Comment by m-arc** [comment]
I agree with this.  Please make app tabs as an option for white listing.
